codec-http2/src/test/java/io/netty/handler/codec/http2/internal/hpack/HuffmanTest.java

```diff
@@ -62,88 +63,103 @@ public void testDecodeEOS() throws IOException {
         for (int i = 0; i < 4; i++) {
             buf[i] = (byte) 0xFF;
         }
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeIllegalPadding() throws IOException {
         byte[] buf = new byte[1];
         buf[0] = 0x00; // '0', invalid padding
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding() throws IOException {
         byte[] buf = makeBuf(0x0f, 0xFF); // '1', 'EOS'
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding1byte() throws IOException {
         byte[] buf = makeBuf(0xFF);
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding2byte() throws IOException {
         byte[] buf = makeBuf(0x1F, 0xFF); // 'a'
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding3byte() throws IOException {
         byte[] buf = makeBuf(0x1F, 0xFF, 0xFF); // 'a'
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding4byte() throws IOException {
         byte[] buf = makeBuf(0x1F, 0xFF, 0xFF, 0xFF); // 'a'
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodeExtraPadding29bit() throws IOException {
         byte[] buf = makeBuf(0xFF, 0x9F, 0xFF, 0xFF, 0xFF);  // '|'
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
     }
 
     @Test(expected = IOException.class)
     public void testDecodePartialSymbol() throws IOException {
         byte[] buf = makeBuf(0x52, 0xBC, 0x30, 0xFF, 0xFF, 0xFF, 0xFF); // " pFA\x00", 31 bits of padding, a.k.a. EOS
-        Huffman.DECODER.decode(buf);
+        decode(new HuffmanDecoder(32), buf);
```

Comment:
nit: consider adding a static method in this class `public static HuffmanDecoder newHuffmanDecoder() { return new HuffmanDecoder(32); }`